app/views/users/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>List of Events you are attending</h1>
+
+<h2>Upcoming events</h2>
+<ul>
+  <% current_user.attended_events.where('date >= ?', DateTime.now).each do |event| %>
```

Comment:
Kindly use the scope from Event model and pass it as an instance variable to your views.
`@upcoming_events = current_user.attended_events.upcoming`